doc.go

```diff
@@ -165,9 +165,35 @@ For HTTP logging, set the GODEBUG environment variable to "http2debug=1" or "htt
 
 Inspecting errors
 
-Most of the errors returned by the generated clients can be converted into a
-`grpc.Status`. Converting your errors to this type can be a useful to get
-more information about what went wrong while debugging.
+Most of the errors returned by the generated clients are wrapped in an
+`apierror.APIError` (https://pkg.go.dev/github.com/googleapis/gax-go/v2/apierror)
+and can be further unwrapped into a `grpc.Status` or `googleapi.Error` depending
+on the transport used to make the call (gRPC or REST). Converting your errors to
+these types can be a useful way to get more information about what went wrong
+while debugging.
+
+Converting to an `apierror.APIError` gives access to specific details in the
```

Comment:
"Converting" in this context sounds to me like an action that can taken by the reader.

Reply:
I'm now sure what you mean - it is something that the user would do?

Reply:
Right, to me this sounds like something the user would do. So maybe just drop "Converting to an ":

```
`apierror.APIError` gives access to specific details in the
```

Reply:
Oh oh gotchya. Done!